src/css/style.css

```diff
@@ -36,54 +36,58 @@ body {
 .header {
   display: flex;
   justify-content: space-between;
-  padding: 0 6.25rem;
-  margin-top: 25px;
+  width: 90%;
+  margin: 0 auto;
+  padding-top: 45px;
 }
 .logo img {
-  width: 140px;
-  height: 120px;
+  width: 230px;
+  object-fit: cover;
 }
 .language {
+  height: 100%;
   display: flex;
+  gap: 15px;
   align-items: center;
 }
-.world {
-  display: flex;
-  align-items: center;
-  margin-right: 10px;
+.world img {
+  object-fit: contain;
 }
-.select-language {
-  display: flex;
-  align-items: center;
+.arrow {
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  background-image: url('../assets/Vector.png');
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-size: 12px;
 }
 .content-select select {
+  padding-right: 30px;
+  padding-left: 30px;
   width: 203px;
   height: 44px;
-  left: 1321px;
-  top: 68px;
-  text-align: center;
   font-size: 18px;
-  background: #ffffff;
+  background-color: #ffffff;
   border: 2px solid #c4c4c4;
-  box-sizing: border-box;
   border-radius: 10px;
-  padding-right: 30px;
-  padding-left: 30px;
-}
+  background-position-x: 93%;
+} 
 .input,
 .fieldset {
   width: 100%;
-  line-height: 16px;
-  margin-top: 0.5em;
-  padding: 1.1em 0.9em;
+  line-height: 65px;
+  margin-top: 6px;
```

Comment:
Comment:
So, I think slowly we are dropping `em` and `rem` in our PRs hehe @e-azocar @victor-heliomar what do you think?

Reply:
math scares me jeje

![download](https://user-images.githubusercontent.com/71513679/174079551-80db2056-8d56-4c12-baf9-c95564e16aad.jpg)

Reply:
@Eghinner I suggest this vscode extension https://marketplace.visualstudio.com/items?itemName=sainoba.px-to-rem

Reply:
I personally prefer to use `rem`, natively `1rem = 16px`, so if I need `8px` is `.5rem`.
I love math, but I you don't want to divide, there's a lot "px to rem" converters.

Reply:
Replace in the attributes that I used. But I'm not sure about replacing the `height` and `width` and the rest in general. In the entire .css

Reply:
@Eghinner 

Is good to change the width and height too: Imagine that you have a Navbar with 40px of height and the text inner of that navbar has a `font-size: 1rem` (16px)

If the user has set her browser with a size basis of 32px (1 rem will be equal to 32px) then practically your Navbar now will not looks good, since the text now is gigant and the navbar height is small for that text jeje (text now have 32px of size, and the navbar 40px, a little more and the text will overflow the navbar)